### PR TITLE
Add a denied namespaces page

### PR DESCRIPTION
**Pull request by CannonLock**
- From here you can delete or approve previously denied namespace if you are an admin

### CODE REVIEW

**Comment by haoming29** [comment]
@CannonLock There's a linter test failed. Could you check that one?

File: web_ui/frontend/components/Namespace/DeniedCard.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                                </>
+                            }
+                        </Box>
+                    </Box>
+                </Box>
+                <Box ref={ref}>
+                    <InformationDropdown adminMetadata={namespace.admin_metadata} transition={transition} parentRef={ref}/>
+                </Box>
+            </Box>
+            <AlertPortal alert={alert} onClose={() => setAlert(undefined)} />
```
Comment (haoming29) [inline web_ui/frontend/components/Namespace/DeniedCard.tsx:154]: I didn't see any alert when I successfully deleted a namespace, but we do if we approve a namespace in the dashboard view. Can we give user a successful message alert here?

**Review by haoming29 — CHANGES_REQUESTED** [review]
One small change requested. LGTM if the linter failure is fixed too

File: web_ui/frontend/components/Namespace/DeniedCard.tsx (hunk 1 above)
Comment (CannonLock) [reply web_ui/frontend/components/Namespace/DeniedCard.tsx]: @haoming29 Going to start transitioning to making success states less intrusive. On success the card now turns green before it disappears rather then also popping up a alert.

**Pull request merged**